docs/diagrams/GradedTestClassDiagram.puml

```diff
@@ -0,0 +1,76 @@
+@startuml GradedTest
```

Comment:
I think it would be good to omit the toString() and hashCode() methods to reduce repeated content

Also once removed, do remember to save an updated UML diagram as .png

Reply:
Noted w thanks :D